main.go

```diff
@@ -246,6 +279,16 @@ func initMetricCollector() {
 		Logger.Infof("collector[%s]: disabled", collectorName)
 	}
 
+	collectorName = "Query"
+	if opts.ScrapeTimeLive.Seconds() > 0 {
+		collectorQueryList[collectorName] = NewCollectorQuery(collectorName, &MetricsCollectorQuery{})
+		collectorQueryList[collectorName].SetAzureProjects(&projectList)
+		collectorQueryList[collectorName].QueryList = opts.QueriesWithProjects
+		collectorQueryList[collectorName].SetScrapeTime(*opts.ScrapeTimeLive)
```

Comment:
Do we really want to scrape that every `30s` by default? Maybe it's better to introduce an own scraping time with default of `30m` (`opts.ScrapeTime`).

There is a rate limit in azure devops.

Reply:
Added ScrapeTimeQuery as you suggested.